setup.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
How does this relate to https://flutter.io/getting-started/ ?

cc @abarth who was working on a new version of that